mate-panel/main.c

```diff
@@ -167,22 +171,17 @@ main (int argc, char **argv)
 	 * connecting to the session manager */
 	panel_session_init ();
 
-    /*Load a css file from a path so the drag handle image can be loaded*/
-    error2 = NULL;
-    provider = gtk_css_provider_new ();
-	        gtk_css_provider_load_from_path (provider,
-				DATADIR "/mate-panel/" "mate-panel.css", &error2);
-
-    if (error2 != NULL) {
```

Comment:
error handling removed!? is that intentional? you can use `parsing-error` signal if you want get errors when loading css.

Reply:
I see that error handling isn't present after every `gtk_builder_add_from_resource` as well...

Reply:
I see that too, but is that needed in our case?, read comment from @lukefromdc

Reply:
and please read that comment from me.
https://github.com/mate-desktop/mate-panel/pull/790#issuecomment-384038849
> No idea to use parsing-error here....
> Help is welcome.

Reply:
So, what about error handling after `gtk_builder_add_from_resource`? Or it's not supposed to fail with GResource...?

Reply:
It can fail, but check documentation:
https://developer.gnome.org/gtk3/stable/GtkBuilder.html#gtk-builder-add-from-resource

> It’s not really reasonable to attempt to handle failures of this call. The only reasonable thing to do when an error is detected is to call g_error().

Reply:
I see, the same is told about `gtk_builder_add_from_file` which was in the code before. Not sure if showing a warning can be considered as error handling... There was a return after that though, without any attempts to continue working with possibly broken widgets.